lib/components/logIn.dart

```diff
@@ -372,63 +321,62 @@ class _LogInScreenState extends State<LogInScreen> {
-                      Expanded(
-                          child: Divider(
-                        color: Colors.grey,
-                      )),
-                      Text(
-                        "Or Connect",
-                        style: TextStyle(color: Colors.grey),
+                    SizedBox(
+                      height: size.height*0.01,
+                    ),
+                    Container(
+                      width: MediaQuery.of(context).size.width - 50,
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Expanded(
+                              child: Divider(
+                            color: Colors.grey,
+                          )),
+                          Text(
+                            "Or Connect",
+                            style: TextStyle(color: Colors.grey),
+                          ),
+                          Expanded(
+                              child: Divider(
+                            color: Colors.grey,
+                          )),
+                        ],
                       ),
-                      Expanded(
-                          child: Divider(
-                        color: Colors.grey,
-                      )),
-                    ],
-                  ),
-                ),
-                SizedBox(height: 15),
-                Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  children: [
-                    GoogleAuthButton(
-                        borderRadius: 29,
-                        iconSize: 30,
-                        style: AuthButtonStyle.icon,
-                        onPressed: () async {
-                          await _auth.signInWithGoogle().then(
-                            (result) async {
-                              if (result != null) {
-                                _image = await urlToFile(_auth.imageUrl);
-                                await uploadProfilePic(context);
-                                Navigator.pushAndRemoveUntil(context,
-                                    MaterialPageRoute(
-                                  builder: (context) {
-                                    return HomeScreen(_auth.userName);
-                                  },
-                                ), (route) => false);
-                              }
-                            },
-                          );
-                        }),
+                    ),
+                    SizedBox(height: size.height*0.01),
+                    Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+                      GoogleAuthButton(
+                          borderRadius: 29,
+                          iconSize: 30,
+                          style: AuthButtonStyle.icon,
+                          onPressed: () async {
+                            var _auth = AuthService();
```

Comment:
This is not how a singleton works, you can see the source code of `Firebase. instance`, 

do this in AuthService and FIrebaseStorageServices class 
```
class AuthSerivce {

AuthService._(); // Private constructor.

final static AuthService _singleton = AuhService._();

static AuthService get instance = _singleton;

}
```
// now in code 

```
 final auth = AuthSerivce.instance; // call only once 


auth.somemethod(username, password);
```

same thing for FirebaseStorage



```

Reply:
i'll do this thanks for explaining